Source/WebKit/Platform/IPC/cocoa/ConnectionCocoa.mm

```diff
@@ -583,17 +610,7 @@ void watchdogTimerFired()
         
         if (m_sendPort) {
             ASSERT(MACH_PORT_VALID(m_receivePort));
-            mach_port_t previousNotificationPort = MACH_PORT_NULL;
```

Comment:
Yes, that's correct, it's the ProcessLauncherCocoa path that is setting one too (and replacing it shouldn't be a problem, since the handler is the same.

I'm 99% confident that we can just remove the callsite in ProcessLauncherCocoa (since the handler gets attached after our new callsite), but there don't appear to be tests for the original issue, so I'm a bit hesitant.